### PR TITLE
Add button to Workshop Admin toolbar linking to User Management

### PR DESCRIPTION
**Pull request by sureshc**
Add button to Workshop Dashboard toolbar linking to PD Workshop User Management feature (/pd/workshop_user_management/facilitator_courses) which currently enables a Workshop Admin to assign Courses to a Facilitator
workshop dashboard button to user management

### CODE REVIEW

**Comment by sureshc** [comment]
@mehalshah @aoby This can wait until I'm back on Wed Sep 6, but would be useful if you have time to review at some point while I'm out next week

**Comment by sureshc** [comment]
It seems like the tests for the 3 different permissions (Organizer, Facilitator, and Workshop Admin) could be combined into a single function that maps the permission to the expected number of buttons and button labels.

**Comment by aoby** [comment]
I like your test improvement suggestion 👍  Go for it.

**Comment by sureshc** [comment]
@mehalshah Can you review the changes I made to the test?

File: apps/test/unit/code-studio/pd/workshop_dashboard/workshop_indexTest.js [hunk 1]
```diff
@@ -18,31 +18,47 @@ describe("WorkshopIndex", () => {
...
-      expect(workshopIndex.find('ButtonToolbar Button').length).to.equal(2);
-      expect(workshopIndex.find('ButtonToolbar Button').map((button) => {
-        return button.children().first().text();
-      })).to.deep.equal(['Facilitator Survey Results', 'Filter View']);
-    });
-
-    it("Organizers and admins have three buttons", () => {
-      ['workshop_organizer', 'workshop_admin'].forEach((permission) => {
+    permissionButtonMap.forEach(function (buttons, permission) {
+      it(permission + " has " + buttons.length + " buttons", () => {
```
Comment (mehalshah) [inline apps/test/unit/code-studio/pd/workshop_dashboard/workshop_indexTest.js:51]: Yay for more efficient tests!

**Pull request merged**